website/docs/r/compute_cluster.html.markdown

```diff
@@ -474,6 +474,12 @@ details, see the referenced link in the above paragraph.
 * `vsan_network_diagnostic_mode_enabled` - (Optional) Enables network
   diagnostic mode for vSAN performance service on the cluster.
 * `vsan_unmap_enabled` - (Optional) Enables vSAN unmap on the cluster.
+* `vsan_remote_datastore_ids` - (Optional) The remote vSAN datastore IDs to be
```

Comment:
Should this mention that [HCI Mesh conflicts with Data-in-Transit](https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vsan.doc/GUID-9113BBD6-5428-4287-9F61-C8C3EE51E07E.html)?